frontend/src/flashcards/FlashcardFront.js

```diff
@@ -16,35 +16,39 @@ const frontStyle={
   display: "flex",
   flexDirection: "column",
   justifyContent: "center",
-  alignItems:"center"
+  alignItems: "center",
 };
 /*
   Styling for the image inside the flashcard
 */
-const imgStyle={
+const imgStyle = {
   display: "block",
   maxWidth: "150px",
-  maxHeight:"150px",
-  width:"auto",
+  maxHeight: "150px",
+  width: "auto",
   height: "auto",
-  margin:"10px",
+  margin: "10px",
 };
 
 class FlashcardFront extends React.Component {
```

Comment:
This component has no state, so you can use a function component instead: https://reactjs.org/docs/components-and-props.html#function-and-class-components.

If you use a function component, it's easier for a code reader to immediately understand that the function has no state.